src/modules/aws-ec2/iam-instance-profile/removeIamInstanceProfile.ts

```diff
@@ -0,0 +1,38 @@
+import { logger } from "../../../logger";
+import { RunbookStep } from "../../RunbookStep";
+import { diassociateIamInstanceProfile } from "./sub-modules/diassociateIamInstanceProfile";
+import { describeIamInstanceProfileAssociations } from "./sub-modules/describeIamInstanceProfileAssociations";
+
+export class RemoveIamInstanceProfile extends RunbookStep {
+  async describeAction(instanceId: string): Promise<void> {
+    const currentValue = await describeIamInstanceProfileAssociations(
+      instanceId
+    );
+
+    if (!currentValue.IamInstanceProfileAssociations) {
+      return;
+    }
+
+    if (currentValue.IamInstanceProfileAssociations.length !== 0) {
+      let profileAssociations = currentValue.IamInstanceProfileAssociations.map(
+        IamInstanceProfileAssociations => {
+          if (!IamInstanceProfileAssociations.IamInstanceProfile) {
+            return "";
```

Comment:
I noticed we had a return with no string in a previous check. Should we be doing the same here?

Reply:
Yep, tested it too. `return` should suffice.